Resources/Prototypes/_Impstation/Voice/speech_emotes.yml

```
@@ -0,0 +1,35 @@
+- type: emote
+  id: Bubble
+  name: chat-emote-name-bubble
+  category: Vocal
+  available: false
+  icon: Interface/Emotes/squish.png
+  whitelist:
+    components:
+    - Vocal
+    - Slime
+  blacklist:
+    components:
+    - BorgChassis
```

You may want to add the `SiliconEmotes` tag to the blacklist to ensure that IPCs do not get this emote unintentionally.
```suggestion
  blacklist:
    tags:
    - SiliconEmotes
    components:
    - BorgChassis
```